File: Python/Ex-3_Teams.py
# 
# Ex-3_Teams.py
# Exercise 3 Explained on Microsoft Teams
# a1127a1fdcff0d741792c01b00b66d7276536d14
#
# Created by Sebastian Jusca on 17/04/2023.
#

F0 = i = 0
F1 = 1

n = int(input(''))

while i < n:
	print(F0)
	Fn = F0 + F1
	F0 = F1
	F1 = Fn
	i += 1
	

File: C/Ex3_Teams.c
//
//  Ex-3_Teams.c
//  Created by Sebastian Jusca on 22/09/2023.
//

#include "stdio.h"

int mcm(int a, int b) {
  int suma = a, sumb = b;
  while (suma != sumb) {
    if (suma < sumb) {suma+=a;}
    else {sumb+=b;}
  }
  return suma;
}

int main(){
  int a, b;

  printf("Enter values: \n");
  scanf("%d",&a);
  scanf("%d",&b);

  printf("%d", mcm(a,b));
  return 0;
}